pkgconf/pkg.yaml

```diff
@@ -0,0 +1,38 @@
+name: pkgconf
+variant: scratch
+dependencies:
+  - stage: base
+  - stage: autoconf
+  - stage: automake
+  - stage: libtool
+steps:
+  - sources:
+      - url: https://github.com/pkgconf/pkgconf/archive/refs/tags/pkgconf-{{ .pkgconf_version }}.tar.gz
+        destination: pkgconf.tar.gz
+        sha256: "{{ .pkgconf_sha256 }}"
+        sha512: "{{ .pkgconf_sha512 }}"
+    prepare:
+      - |
+        tar -xzf pkgconf.tar.gz --strip-components=1
+        autoreconf -vif
+        mkdir build
+        cd build
+
+        ../configure \
+         	--prefix=/usr \
+         	--sysconfdir=/etc \
+         	--mandir=/usr/share/man \
+         	--infodir=/usr/share/info \
+         	--localstatedir=/var \
+         	--with-pkg-config-dir=/usr/local/lib/pkgconfig:/usr/local/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig
```

Comment:
would https://github.com/pkgconf/pkgconf?tab=readme-ov-file#pkgconf-lite work for us?